frontend/src/utils/trapi.js

```diff
@@ -6,42 +6,44 @@
  * @property {Array} results an array of results
  */
 
-/*
+/**
  * Simple graph validity checks
- * Returns list of errors
-*/
-function validateGraph(graph, graphType) {
+ * @param {object} graph - object containing nodes and edges
+ * @param {string} graphName - name of graph for error messages
+ * @returns list of errors
+ */
+function validateGraph(graph, graphName) {
```

Comment:
We could use a JSON schema validator and reference the TRAPI schema.

Reply:
Opened an issue: #185